src/app/views/sidebar/resource-explorer/panels/PathsReview.tsx

```diff
@@ -21,37 +34,138 @@ export interface IPathsReview {
 
 const PathsReview = (props: IPathsReview) => {
   const dispatch: AppDispatch = useDispatch();
-  const { resources: { paths: items } } = useAppSelector(
-    (state) => state
-  );
+  const {
+    resources: { paths: items }
+  } = useAppSelector((state) => state);
   const { isOpen } = props;
-  const headerText = translateMessage('Selected Resources') + ' ' + translateMessage('Preview');
+  const headerText =
+    translateMessage('Selected Resources') + ' ' + translateMessage('Preview');
   const [selectedItems, setSelectedItems] = useState<IResourceLink[]>([]);
+  const [
+    showVersionErrorMessage,
+    {
+      toggle: toggleShowVersionErrorMessage,
+      setTrue: setShowVersionErrorMessageTrue,
+      setFalse: setShowVersionErrorMessageFalse
+    }
+  ] = useBoolean(false);
 
   const columns = [
-    { key: 'url', name: 'URL', fieldName: 'url', minWidth: 300, maxWidth: 350, isResizable: true }
+    {
+      key: 'url',
+      name: 'URL',
+      fieldName: 'url',
+      minWidth: 300,
+      maxWidth: 350,
+      isResizable: true
+    }
   ];
 
+  const getItemsForActions = () => {
+    if (selectedItems.length === 0) {
+      return items;
+    }
+    return selectedItems;
+  };
```

Comment:
When selecting the paths, a user selects the paths that they want to remove from the list. When selection is happening, the Call to action buttons (Generate client & Generate postman collection) are disabled and the remove button is active.
This would remove the need for this function since the items available are the ones the user wants